WooCommerce.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -41,9 +41,9 @@
         "package": "XCUITestHelpers",
         "repositoryURL": "https://github.com/Automattic/XCUITestHelpers",
         "state": {
-          "branch": null,
-          "revision": "47d688b0b2a6356361a16836c4d23f12d6643a73",
-          "version": "0.1.0"
+          "branch": "add-helpers-from-woo",
```

Comment:
Before merging and one https://github.com/Automattic/XCUITestHelpers/pull/3 lands, I'll ship a new version of the framework then point to it here.